build.gradle

```diff
@@ -92,6 +99,15 @@ test {
 shadowJar {
     archiveName = 'RepoSense.jar'
     destinationDir = file("${buildDir}/jar/")
+
+    def repoSenseVersion = project.property("version")
+    if (repoSenseVersion.equals("unspecified")) {
+        repoSenseVersion = versionDetails().lastTag
+    }
```

Comment:
```
    if (repoSenseVersion.equals("unspecified")) {
        if (versionDetails().commitDistance == 0) {
            repoSenseVersion = versionDetails().lastTag
        }
        else {
            repoSenseVersion = versionDetails().gitHash
        }
    }
```
This way the version can be tag or hash. If the `HEAD` is pointing tag, then tag will be used; else the hash will be used.

Reply:
if its a simple change, i think no harm adding that

Reply:
> if its a simple change, i think no harm adding that

This will result that all the netlify preview will have the hash shown in the bottom of report. If it is acceptable, I will start editing the code.

Reply:
Yes that is acceptable to me, thanks @fzdy1914

Reply:
Edited as request